tests/interfaces/test_single_interface.py

```diff
@@ -0,0 +1,103 @@
+import json
+from unittest.mock import patch
+from unittest import TestCase
+
+import responses
+from webapp.app import app
+from webapp.integrations.logic import Interfaces
+
+mock_interface_yaml = """
```

Comment:
This isn't used anywhere, has it been setup for future tests or can we remove it?

Reply:
I can take this out for now.

Reply:
Did you want to remove this in this revision? Not sure if the changes have gone through

Reply:
Yes, it's done now